src/common/asio_misc.h

```diff
@@ -0,0 +1,112 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2018 Red Hat <contact@redhat.com>
+ * Author: Adam C. Emerson <aemerson@redhat.com>
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+#ifndef CEPH_COMMON_ASIO_MISC_H
+#define CEPH_COMMON_ASIO_MISC_H
```

Comment:
since `io_context_pool` is the only thing in `asio_misc.h`, maybe `asio_pool.h` is a better name?

Reply:
Moved to `async/context_pool.h`